src/modules/Dropdown/Dropdown.js

```diff
@@ -98,6 +98,9 @@ export default class Dropdown extends Component {
     /** Name of the hidden input which holds the value. */
     name: PropTypes.string,
 
+    /** Allow user additions to the list of options */
+    allowAdditions: PropTypes.bool,
```

Comment:
This prop requires props `options`, `selection`, and `search`.  See the `selection` propType definition for an example on how to enforce bool while also requiring those other props.